### PR TITLE
chore: add tgi 2.0.0

### PR DESCRIPTION
**Pull request by haixiw**
*Issue #, if available:*

*Description of changes:*
release TGI 2.0.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by amzn-choeric** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: tgi-pr-inf2
* Commit ID: 95f63a34cbac0fe5df91fa3de97ee3073d333b5e
* Result: SUCCEEDED
* [Build Logs](https://cua665myu4.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=0072828e-a381-44fd-995e-fd500de190a9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by amzn-choeric** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: tgi-pr-inf2
* Commit ID: 130b77c095a2ea5a7b472ccb7b10363d5b36b780
* Result: SUCCEEDED
* [Build Logs](https://cua665myu4.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=0097b084-c7da-4bc8-b953-589c07bdce60%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by amzn-choeric** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: tgi-pr-inf2
* Commit ID: 93f15360f5d7f7856d5b6063ab09659737df82bc
* Result: SUCCEEDED
* [Build Logs](https://cua665myu4.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=bc876a76-d2b7-41ed-ab8c-f4ff896db20c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by amzn-choeric** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: tgi-pr-inf2
* Commit ID: a1e2199fd7a79c2830112ebaa45bd3f624fb9b38
* Result: SUCCEEDED
* [Build Logs](https://cua665myu4.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=0a1559c0-2aaa-46f3-b95c-638b77405700%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: releases.json [hunk 1]
```diff
@@ -39,11 +39,12 @@
     "ignore_vulnerabilities": [],
     "releases": [
         {
-            "device": "inf2",
-            "version": "0.0.21",
+            "device": "gpu",
```
Comment (Captainia) [inline releases.json:42]: sanity-check: Is this expected?

**Review by Captainia — APPROVED** [review]

**Comment by haixiw** [comment]
Only minor version updates in dependencis here : https://github.com/huggingface/text-generation-inference/pull/1713 No newly added ones

File: releases.json (hunk 1 above)
Comment (haixiw) [reply releases.json:42]: yeah it's expected. inf2 is https://aws.amazon.com/ec2/instance-types/inf2/ more powerful than normal gpu instances.

**Review by whittech1 — APPROVED** [review]

**Review by akozd — APPROVED** [review]

**Pull request merged**